AltV.Atlas.Boilerplate.Client/Bootstrapper.cs

```diff
@@ -30,20 +33,34 @@ private void ConfigureServices( )
         _serviceCollection.AddTransient<AppSettings>( x => ConfigurationLoader.Load<AppSettings>( "net6.0/appsettings.json" ) );
         
         #region Free Modules
+
         // Register ped module - do the same on server-side and peds module will work :)
         _serviceCollection.RegisterPedModule( );
         _serviceCollection.RegisterVehicleModule( );
+        _serviceCollection.RegisterScaleformModule( );
+        _serviceCollection.RegisterKeyInputModule( );
         #endregion
         
         #region Premium Modules
+
         // _serviceCollection.RegisterPedTrafficModule( ); // https://altv-atlas.github.io/docs/articles/ped-traffic-module.html
+
         #endregion
     }
 
     public void Run( )
     {
+        #region Free Modules
+
+        _serviceProvider.InitializeScaleformModule( );
+        _serviceProvider.InitializeKeyInputModule( new List<Key>{Key.Space, Key.F4} );
```

Comment:
```suggestion
       var whitelistedKeys = new List<Key>{Key.Space, Key.F4};
        _serviceProvider.InitializeKeyInputModule( whitelistedKeys );
```

makes it more clear what that piece of code is doing